5.5/Dockerfile

```diff
@@ -18,14 +18,14 @@ ENV IMAGE_EXPOSE_SERVICES   3306:mysql
 MAINTAINER  Martin Nagy <mnagy@redhat.com>
 EXPOSE 3306
 
-RUN groupadd -g 27 -o -r mysql && \
-    useradd -M -N -g mysql -o -r -d /var/lib/mysql -s /sbin/nologin -c "MySQL Server" -u 27 mysql
-
 RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/mysql55/epel-7-x86_64/download/rhscl-mysql55-epel-7-x86_64.noarch.rpm && \
     yum -y --setopt=tsflags=nodocs install hostname mysql55 && \
     yum clean all
 
+# Make sure the mysql user is installed with UID 27, so we're compatible across volumes.
+RUN test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)"
```

Comment:
@mfojtik yeah i understand your point there, but the reason @mnagy made this change, as I understand it, is because the RPM wants to create the user and requires the user be created w/ UID 27 (whether it creates it or you do).  

I prefer the approach of letting the RPM do it the way it wants rather than us trying to meet its requirements by creating the user manually.  (And then, as @mnagy has, a check so if the behavior ever changes, we catch it)